### DISCUSSION TITLE
GitHub actions on draft PR

### DISCUSSION THREAD

**Discussion by lostystyg** [pocketnetteam/pocketnet.core#163] [Ideas]
@tawmaz @andyoknen  i think we should update actions to do not launch actions for draft pull requests, because they are not really going to be merged but are updated regular and are wasting CI minutes. I did some research and it seems there is and option `if: github.event.pull_request.draft == false` that can be set up for every job and avoid build for draft PRs. 
Or even more it will be even useful to run actions only for PRs that are already going to be merged by using something like:
```code
  pull_request:
    types: [review_requested, ready_for_review]
```
that will also do not allow to run actions on Draft PRs

**Comment by lostystyg** [comment]
E.x. my last push took 23 unnecessary CI minutes

**Comment by andyoknen** [comment]
@lostystyg @tawmaz add types condition d9c1d0f - work.
But this assigns a build task to each specified reviewer..
